plugins/LilyPad/Config.cpp

```diff
@@ -1493,12 +1501,11 @@ INT_PTR CALLBACK DialogProc(HWND hWnd, unsigned int msg, WPARAM wParam, LPARAM l
 					Binding *b;
 					Device *dev;
 					if (GetBinding(port, slot, index, dev, b, ffb)) {
-						const static unsigned int axisUIDs[3] = {UID_AXIS_NEG, UID_AXIS_POS, UID_AXIS};
+						const static int axisUIDs[3] = {UID_AXIS_NEG, UID_AXIS_POS, UID_AXIS};
 						int uid = dev->virtualControls[b->controlIndex].uid;
```

Comment:
Maybe it'd be better to change the UID_AXIS definition from `(1 << 31)` to `(1U << 31)`?

Apart from this line, I think the PR is ok.